src/representations/representation_manager_spherical_invariants.hh

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
This file's changes are just renaming, right?

Reply:
no it has the bispectrum

Reply:
Ummmm.... no. See the addition of the bispectrum, for instance.